flexmeasures/data/schemas/scheduling/__init__.py

```diff
@@ -211,3 +219,91 @@ def _get_variable_quantity_unit(
                 f"Unexpected type '{type(variable_quantity)}' for '{field}': {variable_quantity}."
             )
         return unit
+
+
+class SequentialFlexModelSchema(Schema):
```

Comment:
Currently being used for both simultaneous and sequential multi-asset scheduling, but not meant to be used for the simultaneous case. Misusing it for that is likely causing the double nesting of the `sensor-flex-model` field.

Reply:
The double-nested sensor-flex-model is resolved with https://github.com/FlexMeasures/flexmeasures/pull/1313/commits/56dd63923b6312e0b3b1ac9d055c36b6b4d924d8.